crates/viewer/re_web_viewer_server/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("cargo::rustc-check-cfg=cfg(disable_web_viewer_server)");
```

Comment:
Won't do anything for now, but once we bump to Rust 1.80+ this will tell the checker that this flag actually exists for realsies.